app/templates/signup.hbs

```diff
@@ -1,27 +1,38 @@
 <form class="form">
   <h2>{{this.title}}</h2>
   {{#each this.fields as |field|}}
+  {{#let this.formData as | myFormData |}}
     <FormInput
       @id={{field.id}}
       @for={{field.for}}
       @label={{field.label}}
       @type={{field.type}}
-      @value=''
+      @value={{get this.formData field.id}}
       @errorMessage={{field.errorMessage}}
       @required={{field.required}}
       @showError={{field.showError}}
       @onChange={{this.handleFieldChange}}
-      @validator={{field.validator}}
+      {{!-- @validator={{field.validator}} --}}
     />
```

Comment:
remove comments

Reply:
The piece of code will be required in the future.